### PR TITLE
Customise Profile Page Heading

### LINKED ISSUE #37: Reformat Profile page headings as "USERNAME [ID]"
Reformat Profile page headings as "USERNAME [ID]"

- [ ] Setting @ Profile = Reformat Profile page headings as "USERNAME [ID]"
- [ ] Reformat Profile page headings as "USERNAME [ID]"

### PR DESCRIPTION
**Pull request by Sashank999**
Closes #37 .

### CODE REVIEW

File: extension/scripts/features/customProfileHeading/ttCustomProfileHeading.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	);
+
+	function customHeading() {
+		const skipToContent = document.find("h4#skip-to-content");
+		const onlOrOffIcon = document.find(".profile-container .row.basic-info > *:first-child");
+		const onlOrOffIconClone = onlOrOffIcon.cloneNode(true);
+		const backgroundPos = getComputedStyle(onlOrOffIcon).backgroundPosition;
+		onlOrOffIconClone.style.backgroundPosition = backgroundPos;
+		onlOrOffIconClone.classList.add("tt-profile-icon");
+		skipToContent.innerHTML = `${skipToContent.innerHTML.replace("'s Profile", "")} [${getUserID()}]`;
```
Comment (DeKleineKobini) [inline extension/scripts/features/customProfileHeading/ttCustomProfileHeading.js:26]: This should probably be a different feature.

Comment (DeKleineKobini) [reply extension/scripts/features/customProfileHeading/ttCustomProfileHeading.js:26]: That would also make this one have to change in name.

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]

File: extension/scripts/features/customProfileHeading/ttCustomProfileHeading.js (hunk 1 above)
Comment (Sashank999) [reply extension/scripts/features/customProfileHeading/ttCustomProfileHeading.js:26]: It is actually the same thing.
https://github.com/Mephiles/torntools_extension/blob/master/torntools/scripts/content/profile/ttProfile.js#L1011

Comment (DeKleineKobini) [reply extension/scripts/features/customProfileHeading/ttCustomProfileHeading.js:26]: That code doesn't alter the name?

Settings v5

Comment (Sashank999) [reply extension/scripts/features/customProfileHeading/ttCustomProfileHeading.js]: Done.

File: extension/scripts/features/statusindicator/ttStatusIndicator.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+		const onlOrOffIcon = document.find(".profile-container .row.basic-info > *:first-child");
+		const onlOrOffIconClone = onlOrOffIcon.cloneNode(true);
+		const backgroundPos = getComputedStyle(onlOrOffIcon).backgroundPosition;
+		onlOrOffIconClone.style.backgroundPosition = backgroundPos;
+		onlOrOffIconClone.classList.add("tt-profile-icon");
+		skipToContent.insertAdjacentElement("beforeBegin", onlOrOffIconClone);
+	}
+
+	function removeIndicator() {
+		document.find("h4#skip-to-content").parentElement.find(".tt-profile-icon").remove();
```
Comment (DeKleineKobini) [inline extension/scripts/features/statusindicator/ttStatusIndicator.js:30]: You should probably first check if the icon exists at all.

Comment (Sashank999) [reply extension/scripts/features/statusindicator/ttStatusIndicator.js]: Added check.

**Review by DeKleineKobini — APPROVED** [review]

**Pull request merged**